src/platform/packages/shared/shared-ux/chrome/navigation/README.mdx

```diff
@@ -7,19 +7,36 @@ tags: ['shared-ux', 'component', 'chrome', 'navigation']
 date: 2023-02-28
 ---
 
+# Kibana Chrome Navigation
```

Comment:
If I understand correctly most of this will have to be replaced for the new side nav. basically this is v1, and we will have v2

Reply:
Yes, this package is the entrypoint of rendering React, where `EuiCollapsibleNavBeta` (v1) is actually used.